CHANGELOG.md

```diff
@@ -127,6 +127,95 @@ case it elicits some feedback that requires changes.
 
 * [Video Services](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/video/README.md)
 
+### [Bigtable](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/bigtable/README.md)
+
+We introduced a [new constructor][modern-table-ctor] for `Table` which accepts a
+`DataConnection` instead of a `DataClient`. The `DataConnection` is a new
+interface that more closely matches the client surface of `Table`. Read more
+about `*Connection` classes in our
+[Architecture Design][architecture-connection] document.
+
+**What are the benefits of `DataConnection`?**
```

Comment:
Any reason we should not use `#### What are the benefits of `DataConnection`?

## [Bigtable](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/bigtable/README.md)

Blah

#### What are the benefits of `DataConnection`?

The new API

Reply:
I changed it to use an h4. Although we do things like `**BREAKING CHANGES:** / **Other Changes:**` in h3's.